packages/gated-content/src/conditions/index.ts

```diff
@@ -194,6 +196,19 @@ function toRawSimpleCondition(gqlCondition: gql.ThirdTierCondition): raw.SimpleC
         follow: raw.toProfileId(gqlCondition.follow),
       });
 
+    case 'AdvancedContractCondition':
+      return {
```

Comment:
use the new `raw.advancedContractCondition` builder, it should reduce the boilerplate code here. Don't know if it can solve the readonly. Minor thing anyway.